mypy/config_parser.py

```diff
@@ -318,6 +321,23 @@ def parse_config_file(
             print(f"{file_read}: No [mypy] section in config file", file=stderr)
     else:
         section = parser["mypy"]
+
+        if "files" in section:
+            raw_files = section["files"].strip()
+            files_split = [file.strip() for file in raw_files.split(",")]
+
+            # Remove trailing empty entry if present
+            if files_split and files_split[-1] == "":
+                files_split.pop()
+
+            # Raise an error if there are any remaining empty strings
```

Comment:
Code comments mustn't repeat what's already communicated in code. Their purpose is to provide justification where needed.